oamg/convert2rhel
# [RHELC-1293] Automatically assign pytest markers, ignore warnings

* assign pytest markers automatically
  * https://docs.pytest.org/en/latest/example/markers.html#automatically-adding-markers-based-on-test-names
* ignore PytestUnknownWarning on custom markers starting with test_
  * https://docs.python.org/3/library/warnings.html#warnings.filterwarnings
* remove unused imports
* remove custom markers from both function decorators and pytest.ini
* change some test function names 
* remove a kernel module test verifying, that unloading kernel module will not yield previously appearing inhibitor, as it has litttle to no value




Jira Issues:


- [RHELC-1293](https://issues.redhat.com/browse/RHELC-1293)

Checklist

- [x] PR has been tested manually in a VM (either author or reviewer)
- [x] Jira issue has been made public if possible
- [x] `[RHELC-]` or `[HMS-]` is part of the PR title 
- [x] GitHub label has been added to help with Release notes 
- [x] PR title explains the change from the user's point of view
- [x] Code and tests are documented properly
- [x] The commits are squashed to as few commits as possible (without losing data) 
- [x] When merged: Jira issue has been updated to `Release Pending` if relevant
